test/parallel/test-child-process-fork-regr-gh-2847.js

```diff
@@ -37,7 +37,8 @@ const server = net.createServer(function(s) {
     // is still happening while the server has been closed.
     s.on('error', function(err) {
       if ((err.code !== 'ECONNRESET') &&
-          ((err.code !== 'ECONNREFUSED'))) {
+          (err.code !== 'ECONNREFUSED') &&
+          (err.code !== 'EMFILE')) {
```

Comment:
@refack could you put a brief two line explanation of why this error occurs in a comment above? When reading through tests mysteriously ignored errors are always a bit of code smell.